interface/reports/non_reported.php

```diff
@@ -390,7 +390,7 @@ function tr($a)
             '>
             <span>
               <?php xl('Refresh','e'); ?>
-            </spain>
+            </span>
```

Comment:
This is/was in another PR

Reply:
this is another span tag bug that i Have resolved . I missed this one on the first time.